docs/_config.yml

```diff
@@ -16,7 +16,7 @@
 title: AaC
 description: Enabling DevOps for Model-Based System Engineering
 baseurl: "/" # the subpath of your site, e.g. /blog
-url: "https://jondavid-black.github.io" # the base hostname & protocol for your site, e.g. http://example.com
+url: "https://4000-jondavidblack-aac-s47oo3p955v.ws-us81.gitpod.io/" # the base hostname & protocol for your site, e.g. http://example.com
```

Comment:
Will this be overwritten by the pipeline or something for publishing official docs?

Reply:
Damn, good catch. 😬